app.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
nit: License can be updated to 2020 I believe

Reply:
Good catch! I'll update it to 2020.